aioredis/commands/server.py

```diff
@@ -126,14 +126,26 @@ def debug_segfault(self, key):
         # won't test, this probably works
         return self.execute(b'DEBUG', 'SEGFAULT')  # pragma: no cover
 
-    def flushall(self):
-        """Remove all keys from all databases."""
-        fut = self.execute(b'FLUSHALL')
+    def flushall(self, async=False):
+        """
+        Remove all keys from all databases.
+        :param async: lets the entire dataset to be freed asynchronously. Defaults to False
```

Comment:
Please keep line length less than 80 characters (`make flake` will show this error).
Also add a line break after first sentence to make a separate paragraph for parameter description.